src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Compiler/LoggerPass.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\FrameworkBundle\DependencyInjection\Compiler;
```

Comment:
Would it make sense to move it to HttpKernel as well?

Reply:
done